packages/compass-indexes/src/components/indexes-table/indexes-table.tsx

```diff
@@ -100,53 +128,81 @@ export const IndexesTable: React.FunctionComponent<IndexesTableProps> = ({
     return _columns;
   }, [canDeleteIndex, onSortTable]);
 
+  useEffect(() => {
+    const container =
+      containerRef.current?.getElementsByTagName('table')[0]?.parentElement;
+    if (container) {
+      container.style.height = containerHeight;
+    }
+  }, [containerRef, containerHeight]);
+
+  useEffect(() => {
+    globalAppRegistry.on('compass:compass-shell:resized', () => {
+      setContainerHeight(getContainerHeight(containerRef));
+    });
+  }, [globalAppRegistry, containerRef]);
```

Comment:
Feels a bit drastic that we are making this whole plugin depend on another plugin just for this one header to be sticky.

Reply:
Just wanted to weigh in with a super random comment here:
- having so many indexes to fill an entire screen has a cost, and while several customers may be doing that, it  should not be the case for the majority of them
- this is an optimization that we do not currently have in GA and that can wait
- the content of rows is largely self-explanatory and the table is still usable to a good degree without sticky headers

This is turning out to be a bit hacky, with obscure interactions and multiple plugins involved. If we want to go on with it I'm happy to reconsider it but I feel in this case we may rather want to stay patient, open a ticket for LG and wait for a proper feature on the table component itself, so we avoid creating more troubles for ourselves than those we solve.

Reply:
Also of course, scrap that if there is a clean solution instead

Reply:
i just cleaned it up and used AutoResizer in this one. However, we can also reach out to LG for adding fixed header support.